components/edit-collective/sections/PaymentReceipts.js

```diff
@@ -0,0 +1,270 @@
+import LoadingPlaceholder from '../../LoadingPlaceholder';
+import MessageBoxGraphqlError from '../../MessageBoxGraphqlError';
+import StyledButton from '../../StyledButton';
+import StyledCard from '../../StyledCard';
+import StyledSelect from '../../StyledSelect';
+import { H2, H3, P, Span } from '../../Text';
+
+const Divider = styled(Box)`
+  border-bottom-width: 1px;
+  border-bottom-style: ${props => props.borderStyle || 'solid'};
+  border-bottom-color: ${props => props.borderColor || '#C4C7CC'};
+`;
+
+const HostName = styled(P)`
+  margin: 0 !important;
+`;
+
+dayjs.extend(utc);
+
+export const invoicesQuery = gql`
+  query TransactionsDownloadInvoices($fromCollectiveSlug: String!) {
+    allInvoices(fromCollectiveSlug: $fromCollectiveSlug) {
+      slug
+      year
+      month
+      totalAmount
+      totalTransactions
+      currency
+      fromCollective {
+        slug
+      }
+      host {
+        slug
+        name
+        imageUrl
+      }
+    }
+  }
+`;
+
+const filterInvoices = (allInvoices, filterBy) => {
+  if (filterBy === 'past_12_months') {
+    const twelveMonthsAgo = dayjs().subtract(11, 'month');
+    return allInvoices.filter(i => {
+      const dateMonth = dayjs.utc(`${i.year}-${i.month}`, 'YYYY-M');
+      return dateMonth.isAfter(twelveMonthsAgo);
+    });
+  }
+
+  return allInvoices.filter(i => i.year === filterBy);
+};
+
+const ReceiptsLoadingPlaceholder = () => (
+  <Flex flexDirection="column">
+    <Flex alignItems="center" justifyContent="space-between">
+      <LoadingPlaceholder mr={3} width="104px" height="24px" />
+      <Divider width="80%" borderBottom="1px solid #C4C7CC" />
+    </Flex>
+    {Array.from({ length: 3 }, (_, index) => (
+      <StyledCard my={3} key={index} display="flex" alignItems="center" py={3} px="24px">
+        <LoadingPlaceholder borderRadius="16px" width="48px" height="48px" mr={3} />
+        <Box>
+          <LoadingPlaceholder mb={2} width={['164px', '361px']} height={['40px', '24px']} />
+          <LoadingPlaceholder width="115px" height="14px" />
+        </Box>
+      </StyledCard>
+    ))}
+  </Flex>
+);
+
+const NoReceipts = () => (
+  <Flex alignItems="center" justifyContent="center" my={5}>
+    <StyledCard height="100px" padding="16px 24px" display="flex" alignItems="center" justifyContent="center">
+      <H3 fontSize="15px" lineHeight="24px" color="black.500" textAlign="center">
+        <FormattedMessage id="paymentReceipt.noReceipts" defaultMessage="No receipt available within this period." />
+      </H3>
+    </StyledCard>
+  </Flex>
+);
+
+const renderReceiptCard = (invoice, index) => (
+  <StyledCard
+    my={3}
+    key={index.toString()}
+    alignItems="center"
+    display="flex"
+    flexDirection={['column', 'row']}
+    justifyContent="space-between"
+    py={3}
+    px="24px"
+  >
+    <Flex alignItems="center">
+      <Avatar collective={invoice.host} borderRadius="16px" mr={3} size="48px" />
+      <Box>
+        <HostName
```

Comment:
I tried but apparently, we have https://github.com/opencollective/opencollective-frontend/blob/master/components/Body.js#L74 taking precedence over `p` margin property. I don't want to remove it because a lot of components might probably be relying on it.